fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java

```diff
@@ -358,11 +461,18 @@ private boolean relocateAndBalance(GroupId groupId, Set<Long> unavailableBeIds,
                 }
             }
 
+            if (!hasUnavailableBe && !unavailableBeIds.isEmpty() && isChanged) {
```

Comment:
hasUnavailableBe or !hasUnavailableBe?
!hasUnavailableBe means all the be is available.

Reply:
`!hasUnavailableBe` is right condition
1. `!hasUnavailableBe` means that we have already handled all the unavailable backends
2. `!unavailableBeIds.isEmpty()` means that there exists unvailable backend which we have to handle
3. `isChanged` means that we have changed the backend set because of unavailable backends.

btw: i have tested this in the unit test.